composer.json

```diff
@@ -29,10 +29,10 @@
     "require-dev": {
         "php-parallel-lint/php-parallel-lint": "1.4.0",
         "phpstan/extension-installer": "1.4.3",
-        "phpstan/phpstan": "1.12.9",
-        "phpstan/phpstan-phpunit": "1.4.0",
+        "phpstan/phpstan": "2.0.3",
+        "phpstan/phpstan-phpunit": "2.0.1",
```

Comment:
I think we need an OR-list of versions, e.g. `"phpstan/phpstan": ".12.0 || 2.0.3"`.  That way we can have the new version installed when the PHP environment allows, whilst also continuing to support older PHP versions.